MaxText/scratch_code/golden_gemma2-2b_export-flax.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
Do we want to open source all the notebooks?

Reply:
I think we had the original [gemma export notebook](https://github.com/google/maxtext/blob/main/MaxText/scratch_code/golden_gemma-2b_export.ipynb) and other notebooks (llama, mixtral, etc) open sourced so added the gemma2 notebook to scratch_code folder as well with other open source notebooks